pycheribuild/config/chericonfig.py

```diff
@@ -409,7 +409,8 @@ def load(self):
     def _initialize_derived_paths(self):
         # Set CHERI_BITS variable to allow e.g. { cheribsd": { "install-directory": "~/rootfs${CHERI_BITS}" } }
         os.environ["CHERI_BITS"] = self.mips_cheri_bits_str
-        os.environ["CHERI_CAPTABLE_ABI"] = self.cheri_cap_table_abi
+        if self.cheri_cap_table_abi:
```

Comment:
This can probably go completely. I believe I only added it to allow different install/build dirs in the config file using 
`"install-directory": "/a/b/foo-$CHERI_CAPTABLE_ABI"`. But that's done automatically now.